api/cityNameSlice.ts

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from "@reduxjs/toolkit";
+import type { PayloadAction } from '@reduxjs/toolkit';
+
+export interface cityState {
+  value: string
+}
+
+const initialState: cityState = {
+  value: "",
```

Comment:
We may want to name this something more descriptive.  What does this value represent?  The city?  Zip code?  Let's try to make this obvious.

Reply:
value was changed to cityName